apps/utils/scripts/run-bfd-server

```diff
@@ -174,7 +174,7 @@ function setvar() {
   else
     full_ssm_name="${SSM_PARAMETER_PATH}/${ssm_name}"
     echo put-parameter $full_ssm_name
-    aws ssm put-parameter --overwrite --endpoint-url http://localhost:4566 --name "${full_ssm_name}" --value "${value}" # > /dev/null
+    aws ssm put-parameter --overwrite --endpoint-url http://localhost:4566 --name "${full_ssm_name}" --value "${value}" --no-cli-pager # > /dev/null
```

Comment:
These script changes are unrelated, but having to confirm the SSM parameter updates every time was getting very annoying. Adding `--no-cli-pager` just removes the need to confirm the parameter updates every time it runs.

Reply:
This makes sense and helps avoid the _works on my machine_ commentary. I know on _my machine_, my `~/.aws/config` includes a blanked value for this under default, e.g.

```
[default]
cli_pager=
```

Reply:
Ah, makes sense. I didn't know there was a setting for this.